project.yaml

```diff
@@ -4,146 +4,146 @@ targets:
       x86_64:
         test_distros:
           representative:
-            debian-12
+          - debian-12
       aarch64:
         test_distros:
           representative:
-            debian-12-arm
+          - debian-12-arm
   bullseye:
     architectures:
       x86_64:
         test_distros:
           representative:
-            debian-11
+          - debian-11
       aarch64:
         test_distros:
           representative:
-            debian-11-arm
+          - debian-11-arm
   buster:
     architectures:
       x86_64:
         test_distros:
           representative:
-            debian-10
+          - debian-10
   centos7:
     architectures:
       x86_64:
         test_distros:
           representative:
-            centos-7
+          - centos-7
           exhaustive:
-            rhel-7
-            rhel-7-9-sap-ha
+          - rhel-7
+          - rhel-7-9-sap-ha
   focal:
     architectures:
       x86_64:
         test_distros:
           representative:
-            ubuntu-minimal-2004-lts
+          - ubuntu-minimal-2004-lts
           exhaustive:
-            ubuntu-2004-lts
+          - ubuntu-2004-lts
       aarch64:
         test_distros:
           representative:
-            ubuntu-minimal-2004-lts-arm64
+          - ubuntu-minimal-2004-lts-arm64
           exhaustive:
-            ubuntu-2004-lts-arm64
+          - ubuntu-2004-lts-arm64
   jammy:
     architectures:
       x86_64:
```

Comment:
should architectures be in a list too? seems like yes to me, but I dunno

Reply:
Lists can't be indexed by key, only integers. this would break the logic we have.